app/models/user.rb

```diff
@@ -1,12 +1,14 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- [ ] Good job adding devise to your project. However, as stated in the requirements, please ensure to add the feature to **ask for email confirmation**.
To accomplish this task, you need to enable **:confirmable**  in the user model. Like below :point_down: 
```
  devise :database_authenticatable, :registerable,
         :recoverable, :rememberable, :validatable, :confirmable
```

#### Requirement :point_down: 
![Screenshot from 2023-05-19 19-11-15](https://github.com/Chaseknis/blog_app/assets/89228743/02394f5c-17f6-4fff-ac58-9ea2b5916e65)